build/__phpVersions.sh

```diff
@@ -7,7 +7,7 @@ COMPOSER_SETUP_SHA384='906a84df04cea2aa72f40b5f787e49f22d4c2f19492ac310e8cba5b96
 COMPOSER1_9_VERSION='1.9.3'
 COMPOSER1_10_VERSION='1.10.19'
 COMPOSER2_0_VERSION='2.0.8'
-COMPOSER2_2_VERSION='2.2.9'
+COMPOSER2_2_VERSION='2.3.4'
```

Comment:
Could you please confirm at which point we're updating the binaries? Is it just that this PR is in progress and you will add it in another iteration? Similar question for the tests, that I imagine you'll just add later

Reply:
I'm working on updating the binaries now. But I have moved to `draft` for now until I get some testing added.

Reply:
Sounds good, thanks

Reply:
Test has been added. Moving away from draft.